.github/ISSUE_TEMPLATE/build-a-component.md

```diff
@@ -0,0 +1,57 @@
+---
+name: "Build a component"
+about: Everything we need to do when we build a component
+title: ''
+labels: "contribution or major iteration, design, epic"
+assignees: ''
+
+---
+
+## What
+<!-- Add name of component, plus any relevant links (for example, community backlog issues and comments, pull requests for the component and its guidance) -->
+
+## Why
+<!-- Add reason for building the component -->
+
+## Who needs to know about this
+<!-- Add team roles involved in building the component -->
+
+## Acceptance criteria
+<!-- Customise, and add component-specific checklist items to, this list of criteria all components need to meet -->
+### Accessibility
+In general, the component must:
+- [ ] work with [assistive technologies listed in the service manual(https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices)
+- [ ] pass tests on tools like axe and HTML validator
+- [ ] works when zoomed at high resolution
+- [ ] work in high contrast mode
+- [ ] follow [focus state guidance](https://design-system.service.gov.uk/get-started/focus-states/)
+
+For more information on accessibility testing, [read our documentation on testing components](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/test-components-using-accessibility-acceptance-criteria.md).
+
+### Coding standards
+The component's guidance must include these sections:
+- [ ] have Nunjucks options that are consistent with other Design System components
+- [ ] have commonly available Nunjuck options
+- [ ] follow GOV.UK Frontend coding conventions
+- [ ] follow the code style guide
+
+### Content
+The component must:
+- [ ] include Nunjucks documentation
+- [ ] have PR commits with [commit messages that are easy to understand](https://github.com/alphagov/design-system-team-docs/blob/main/development/commit-messages.md)
+
+### Reviews
+The component must:
+- [ ] be reviewed by the working group at the development stage
+- [ ] in its final version, receive a design crit from the wider community, to see if we've balanced any trade-offs correctly
+
+### Tests
+The component must:
+- [ ] avoid a breaking change
```

Comment:
You're right, I think this is a balancing act rather than a hard rule. Where possible, we avoid breaking changes. But  sometimes (for example, with the accordion component iteration) a breaking change is necessary.

Is this template solely about building new components, or do we think it applies to major iterations too? It's less likely that a new component will be a breaking change (unless we were splitting existing components, for instance), but it's more likely with iterations.

Reply:
I think this is just for new components. Maybe in due course we can have similar templates for major iterations, but tweaked to reflect differences like these.